build.sh

```diff
@@ -138,6 +138,22 @@ else
     echo -e "Warning: Installation script couldn't detect host OS..\n" # exit ?
 fi
 
+SET_CUSTOM_ICU() {
+    ICU_PATH=$1
+    if [[ ! -d "$ICU_PATH" || ! -d "$ICU_PATH/unicode" ]]; then
+        ICU_PATH_LOCAL="$CHAKRACORE_DIR/$ICU_PATH"
+        if [[ ! -d "$ICU_PATH_LOCAL" || ! -d "$ICU_PATH_LOCAL/unicode" ]]; then
+            # if a path was explicitly provided, do not fallback to no-icu
+            echo "!!! couldn't find ICU at either $ICU_PATH or $ICU_PATH_LOCAL"
+            exit 1
+        else
+            ICU_PATH="$ICU_PATH_LOCAL"
+        fi
+    fi
+    CMAKE_ICU="-DICU_INCLUDE_PATH_SH=$ICU_PATH"
```

Comment:
`CMAKE_ICU="-DICU_INCLUDE_PATH_SH=${ICU_PATH}"` instead?

Reply:
any particular reason?

Reply:
old habit. In such context `${}` is less error-phrone